Web Quiz Engine (Kotlin)/task/src/engine/entity/Quiz.kt

```diff
@@ -3,16 +3,24 @@ package engine.entity
 import com.fasterxml.jackson.annotation.JsonInclude
 import com.fasterxml.jackson.annotation.JsonProperty
 import engine.QuizIdentifier
+import jakarta.persistence.*
 import jakarta.validation.constraints.NotBlank
 import jakarta.validation.constraints.Size
 
+@Entity
 data class Quiz(
+    @Id
+    @GeneratedValue
     val id: QuizIdentifier? = null,
     @field:NotBlank(message = "The quiz must have a title")
     val title: String,
     @field:NotBlank(message = "The quiz must have a text")
     val text: String,
     @field:Size(min = 2, message = "The quiz must have at least 2 options")
+    @ElementCollection(fetch = FetchType.EAGER)
     val options: List<String>,
-    @JsonInclude(JsonInclude.Include.NON_NULL) @JsonProperty("answer") val answerList: List<Int>? = null,
+    @ElementCollection(fetch = FetchType.EAGER)
+    @JsonInclude(JsonInclude.Include.NON_NULL)
```

Comment:
I don't think these two Json annotations are doing anything.
The fasterxml jackson annotations are more for REST contract validations and payload format, not for entity persistence.

Reply:
Ahhhh. Now I see that you are actually using this Object in your Controller so they ARE doing something.

Reply:
hmmmm, does that mean there is something I should separate or no changes are required?

Reply:
See comment: https://github.com/f8full/hyperskill-web-quiz-engine/pull/2#discussion_r1564720501

Reply:
I think this is linked so consider my question answered.
https://github.com/f8full/hyperskill-web-quiz-engine/pull/2#discussion_r1564737432

Reply:
Done
https://github.com/f8full/hyperskill-web-quiz-engine/pull/2#discussion_r1564931453